packages/editor/src/lib/editor/Editor.ts

```diff
@@ -720,12 +720,12 @@ export class Editor extends EventEmitter<TLEventMap> {
 			throw Error(`No state found for initialState "${initialState}".`)
 		}
 
-		this.root.enter(undefined, 'initial')
-
 		this.edgeScrollManager = new EdgeScrollManager(this)
 		this.focusManager = new FocusManager(this, autoFocus)
 		this.disposables.add(this.focusManager.dispose.bind(this.focusManager))
 
+		this.root.start()
```

Comment:
Previously this was an `enter()`, but this does enough different that it should be its own method.